certs_wallet/src/chaincert.rs

```diff
@@ -80,6 +80,51 @@ pub(crate) fn deposit_chaincert(
     write_chaincerts(env, &chaincerts)
 }
 
+pub(crate) fn revoke_chaincert(
+    env: &Env,
+    chaincert_id: &Bytes,
+    contract_distributor: &Address,
```

Comment:
Based on the previous Pull Request, should we use the same name for this argument? `distributor_contract`